app/settings.py

```diff
@@ -5,7 +5,8 @@
 from requests.adapters import HTTPAdapter
 
 
-LOGGING_LEVEL = logging.getLevelName(os.getenv('LOGGING_LEVEL', 'DEBUG'))
+LOGGING_LEVEL = logging.getLevelName(os.getenv("LOGGING_LEVEL", "DEBUG"))
+LOGGING_FORMAT = "%(asctime)s.%(msecs)06dZ|%(levelname)s: sdx-collect: %(message)s"
```

Comment:
The date format is set in main.py. 'datefmt="%Y-%m-%dT%H:%M:%S"' does not have milliseconds.